src/Registry/lib/Registry/TrackHub/Validator.pm

```diff
@@ -56,7 +57,9 @@ sub new {
 sub validate {
   my ($self, $file) = @_;
 
-  my $cmd = sprintf("validate.py -s %s -f %s", $self->{schema}, $file);
+  my $cfile = __FILE__;
+  my ($name, $path, $suffix) = File::Basename::fileparse($cfile);
+  my $cmd = sprintf("$path/../../../../../docs/trackhub-schema/validate.py -s %s -f %s", $self->{schema}, $file);
```

Comment:
I would suggest moving first the validation script to a more appropriate location, e.g. the script directory under root.